app/src/features/FhirResourceTree/useGetSelectedAttributeAncestors.ts

```diff
@@ -0,0 +1,47 @@
+import { useMemo } from "react";
+
+import { useParams } from "react-router";
+
+import { useApiAttributesRetrieveQuery } from "services/api/endpoints";
+
+import { computePathWithoutIndexes } from "./resourceTreeUtils";
+
+/**
+ * `Observation.code.coding[0].type` => `[Observation.code, Observation.code.coding, Observation.code.coding[0]]`
+ * @param path Path to decompose
+ * @returns Ancestors paths
+ */
+const getAncestorsPaths = (path: string): string[] => {
+  return path.split(".").reduce((acc: string[], val, index, array) => {
```

Comment:
nit: when you have a single `return` statement in the function, don't use brackets/return